lab_3_keywords_textrank/main.py

```diff
@@ -136,7 +148,16 @@ def encode(self, tokens: tuple[str, ...]) -> Optional[tuple[int, ...]]:
                 sequence of integer tokens
         In case of empty tokens input data, None is returned
         """
-        pass
+        if not tokens:
+            return None
+
+        self._learn_indices(tokens)
+        numbers_tokens = []
+        for token in tokens:
+            if token in self._word2id:
+                numbers_tokens.append(self._word2id.get(token))
```

Comment:
You don't need to use the `get` method if you check that the `token` is in the `self._word2id`

Reply:
fixed